scanners/google_test.go

```diff
@@ -246,8 +246,8 @@ func TestGoogleSearchScan(t *testing.T) {
 			expectedResult := []*GoogleSearchDork{
 				{
 					Number: "+33673421322",
-					Dork:   "site:facebook.com intext:\"33673421322\" OR intext:\"+33673421322\" OR intext:\"0673421322\"",
-					URL:    "https://www.google.com/search?q=site%3Afacebook.com+intext%3A%2233673421322%22+OR+intext%3A%22%2B33673421322%22+OR+intext%3A%220673421322%22",
+					Dork:   "site:facebook.com intext:\"33673421322\" OR intext:\"+33673421322\"  OR intext:\"06 73 42 13 22\" OR intext:\"0673421322\"",
```

Comment:
There's a typo in your patch, you added an extra space before `OR intext:\"06 73 42 13 22\"`